cpython.proj

```diff
@@ -0,0 +1,178 @@
+<Project Sdk="Microsoft.Build.Traversal">
+
+  <Target Name="ProvisionUniversalOpenSSL" BeforeTargets="BuildPythonUnix" Condition="'$(ProvisionUniversalOpenSSL)' == 'true' and !Exists('$(ProvisionedOpenSSLDir)\include\openssl\ssl.h')">
+    <!-- Python needs to build against a static OpenSSL, in order for https to work (e.g.
+         if pip is expected to work). Unfortunately, we're expected to take OpenSSL from
+         Homebrew, and Homebrew does not in any way shape or form support Universal
+         binaries, or alt prefix installation. In order to build a Universal Python, we
+         need a Universal libssl - so we create one ourselves, by downloading two copies
+         into the Homebrew package cache, exctract the tarballs ourselves to avoid the need
+         for two copies of Homebrew, then combine the two together with lipo. -->
+    <DownloadFile DestinationFileName="brew.tar.gz"
+      DestinationFolder="$(BaseIntermediateOutputPath)"
+      Retries="10"
+      SourceUrl="https://github.com/Homebrew/brew/tarball/master" />
```

Comment:
this whole thing still makes me uneasy, but please use at least a commit instead of `master`.